client/flags/flags.go

```diff
@@ -91,6 +94,8 @@ var LineBreak = &cobra.Command{Run: func(*cobra.Command, []string) {}}
 // AddQueryFlagsToCmd adds common flags to a module query command.
 func AddQueryFlagsToCmd(cmd *cobra.Command) {
 	cmd.Flags().String(FlagNode, "tcp://localhost:26657", "<host>:<port> to Tendermint RPC interface for this chain")
+	cmd.Flags().String(FlagGRPCNode, "", "<host>:<port> to GRPC interface for this chain")
```

Comment:
@troian I understand your point about grpc URI deriving, but imo it's a bit confusing. What do you think about this:

```suggestion
	cmd.Flags().String(FlagGRPCNode, "localhost:9090", "<host>:<port> to GRPC interface for this chain")
```

The UX is:
- by default, the client.Context pings localhost:9090
- if the user wants to change gRPC uri, then `--grpc-node=<...>` or `_GRPC_NODE=<...>` (both are equivalent)
- if the user explicitly wants to use Tendermint rpc, then `--grpc-node=""` (and optionally `--node=<...>`)